test/basic.tests.js

```diff
@@ -592,79 +587,31 @@ contract('LiquidityBridgeContract', async accounts => {
         expect(contractBalanceBefore.toString()).to.be.eq(contractBalanceAfter.toString());
     });
 
-    it('should verify signature for pegout and fail hash because Bridge address is not valid as destination', async () => {
```

Comment:
why removing these 2 tests?

Reply:
Was removed because they are related to the bridge address.